### PR TITLE
Fix formatting cumulative memory in Web UI

### PR DESCRIPTION
**Pull request by losipiuk**
Value for cumulative memory of a query was previously reported using
following format:

"4.33T seconds". It is not obviouls that it represents integer of memory
coinsumption over time.
This commit changes format to:
"4.33TB*seconds"

## Documentation

(x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
(x) Release notes entries required with the following suggested text:

```markdown
# Web UI
* Improve readability of Cumulative User Memory on Query Details page. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by losipiuk** [comment]
image

**Review by radek-starburst — APPROVED** [review]

**Comment by losipiuk** [comment]
Previously: 

image

**Comment by findepi** [comment]
Can you please make labeller apply the `ui` label?

File: core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx [hunk 1]
```diff
@@ -1502,11 +1502,11 @@ export class QueryDetail extends React.Component {
                                             Cumulative User Memory
                                         </td>
                                         <td className="info-text">
-                                            {formatDataSizeBytes(query.queryStats.cumulativeUserMemory / 1000.0) + " seconds"}
+                                            {formatDataSize(query.queryStats.cumulativeUserMemory / 1000.0) + "*seconds"}
                                         </td>
                                         {taskRetriesEnabled &&
                                         <td className="info-failed">
-                                            {formatDataSizeBytes(query.queryStats.failedCumulativeUserMemory / 1000.0) + " seconds"}
+                                            {formatDataSize(query.queryStats.failedCumulativeUserMemory / 1000.0) + "*seconds"}
```
Comment (findepi) [inline core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx:1509]: `MB*s` ? (`MBs` would be correct, but not clear)

**Review by findepi — APPROVED** [review]
optional

File: core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx (hunk 1 above)
Comment (losipiuk) [reply core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx:1509]: I started with MBs but yeah, I think it is not clear. 
Elsewhere we use full `seconds` (not an abbreviation), so I guess this is fine.

**Comment by losipiuk** [comment]
> Can you please make labeller apply the `ui` label?

👍

**Pull request merged**